src/commands/admin/removecourse.ts

```diff
@@ -72,6 +72,8 @@ export default class extends Command {
 					await interaction.guild.members.fetch();
 					const staffRole = await interaction.guild.roles.cache.find(role => role.name === `${courseId} Staff`);
 					const studentRole = await interaction.guild.roles.cache.find(role => role.name === `CISC ${courseId}`);
+					const allStaffRole = await interaction.guild.roles.cache.find(role => role.name === 'Staff');
+					const profRole = await interaction.guild.roles.cache.find(role => role.name === 'Prof');

```

Comment:
Do we also want to remove the TA/LA role from student staff?

Reply:
Definitely, yeah